.code-samples.meilisearch.yaml

```diff
@@ -76,16 +76,62 @@ search_post_1: |-
 search_get_1: |-
   curl \
     -X GET 'http://localhost:7700/indexes/movies/search?q=american%20ninja'
-get_update_1: |-
+get_task_by_index_1: |-
   curl \
-    -X GET 'http://localhost:7700/indexes/movies/updates/1'
-get_all_updates_1: |-
+    -X GET 'http://localhost:7700/indexes/movies/tasks/1'
+get_all_tasks_1: |-
   curl \
-    -X GET 'http://localhost:7700/indexes/movies/updates'
-get_keys_1: |-
+    -X GET 'http://localhost:7700/tasks'
+get_task_1: |-
   curl \
-    -H "X-Meili-API-Key: 123" \
-    -X GET 'http://localhost:7700/keys'
+    -X GET 'http://localhost:7700/tasks/1'
+get_all_tasks_by_index_1: |-
+  curl \
+    -X GET 'http://localhost:7700/indexes/movies/tasks'
+get_one_key_1: |-
+  curl \
+    -X GET 'http://localhost:7700/keys/d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4' \
+    -H 'Authorization: Bearer masterKey'
+get_all_keys_1: |-
+  curl \
+    -X GET 'http://localhost:7700/keys' \
+    -H 'Authorization: Bearer masterKey'
+create_a_key_1: |-
+  curl \
+    -X POST 'http://localhost:7700/keys' \
+    -H 'Authorization: Bearer masterKey' \
+    -H 'Content-Type: application/json' \
+    --data-binary '{
+      "description": "Add documents: Products API key",
+      "actions": [
+        "documents.add"
+      ],
+      "indexes": [
+      "products"
+      ],
+      "expiresAt": "2021-11-13T00:00:00Z"
```

Comment:
This date has passed so if the user tries this command it will return an error.

Reply:
Hmmm…that's true. The fact is that any date will eventually generate an error, unless we go for something completely unreasonable like 2121-11-13. What do you think, @dichotommy?

Reply:
Is there a way of interpolating the date with vuepress? Like, instead of writing `2021-11-13` we write something like `{{currentYear}}-11-13`?

Reply:
To answer my own question, vuepress allows doing something like that, but it doesn't work within code samples (likely because code samples are not parsed the same way as regular content).

Reply:
> Is there a way of interpolating the date with vuepress?

Perhaps, but it strikes me as an overly complex solution. @bidoubiwa , thoughts since you developed the plugin?

I would suggest we change this to some date within the next hundred years, ideally one with some kind of significance or joke attached so it's more obvious that users shouldn't pay too much attention to it.

Reply:
you're probably right. Let's go with that.